File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/seasons/EpisodeDetails.java
package com.rkbapps.neetflix.models.tvseries.seasons;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.rkbapps.neetflix.models.castandcrew.Cast;

import java.io.Serializable;
import java.util.List;


public class EpisodeDetails implements Serializable {
    @SerializedName("air_date")
    @Expose
    private String airDate;
    @SerializedName("episode_number")
    @Expose
    private Integer episodeNumber;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("overview")
    @Expose
    private String overview;
    @SerializedName("runtime")
    @Expose
    private Integer runtime;
    @SerializedName("season_number")
    @Expose
    private Integer seasonNumber;
    @SerializedName("show_id")
    @Expose
    private Integer showId;
    @SerializedName("still_path")
    @Expose
    private String stillPath;
    @SerializedName("vote_average")
    @Expose
    private Double voteAverage;
    @SerializedName("vote_count")
    @Expose
    private Integer voteCount;
    @SerializedName("guest_stars")
    @Expose
    private List<Cast> guestStars;

    public String getAirDate() {
        return airDate;
    }

    public Integer getEpisodeNumber() {
        return episodeNumber;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getOverview() {
        return overview;
    }

    public Integer getRuntime() {
        return runtime;
    }

    public Integer getSeasonNumber() {
        return seasonNumber;
    }

    public Integer getShowId() {
        return showId;
    }

    public String getStillPath() {
        return stillPath;
    }

    public Double getVoteAverage() {
        return voteAverage;
    }

    public Integer getVoteCount() {
        return voteCount;
    }

    public List<Cast> getGuestStars() {
        return guestStars;
    }

    @Override
    public String toString() {
        return "EpisodeDetails{" +
                "airDate='" + airDate + '\'' +
                ", episodeNumber=" + episodeNumber +
                ", id=" + id +
                ", name='" + name + '\'' +
                ", overview='" + overview + '\'' +
                ", runtime=" + runtime +
                ", seasonNumber=" + seasonNumber +
                ", showId=" + showId +
                ", stillPath='" + stillPath + '\'' +
                ", voteAverage=" + voteAverage +
                ", voteCount=" + voteCount +
                ", guestStars=" + guestStars +
                '}';
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/seasons/EpisodeDetails.kt
package com.rkbapps.neetflix.models.tvseries.seasons

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.rkbapps.neetflix.models.castandcrew.Cast
import java.io.Serializable

data class EpisodeDetails(
    @SerializedName("air_date")
    @Expose
    val airDate: String,

    @SerializedName("episode_number")
    @Expose
    val episodeNumber: Int,

    @SerializedName("id")
    @Expose
    val id: Int,
    @SerializedName("name")
    @Expose
    val name: String,

    @SerializedName("overview")
    @Expose
    val overview: String,

    @SerializedName("runtime")
    @Expose
    val runtime: Int,

    @SerializedName("season_number")
    @Expose
    val seasonNumber: Int,

    @SerializedName("show_id")
    @Expose
    val showId: Int,

    @SerializedName("still_path")
    @Expose
    val stillPath: String?,

    @SerializedName("vote_average")
    @Expose
    val voteAverage: Double,

    @SerializedName("vote_count")
    @Expose
    val voteCount: Int,

    @SerializedName("guest_stars")
    @Expose
    val guestStars: List<Cast>,

    ) : Serializable